### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

### CODE REVIEW

**Comment by kianmeng** [comment]
@keepcosmos Any updates on this?

**Comment by Valian** [comment]
Disclaimer: A few days ago I became a maintainer of this project. I'm planning to release a new version soon.

@kianmeng You created a very nice PR that I'll be reviewing tomorrow. In the meantime, any chance you could update mix.exs and mix.lock so I can merge it? 👍

File: mix.exs [hunk 1]
```diff
@@ -67,13 +50,25 @@ defmodule Readability.Mixfile do
 
   defp package do
     [
+      description: "Readability library for extracting and curating articles.",
       files: ["lib", "mix.exs", "README*", "LICENSE*", "doc"],
       maintainers: ["Jaehyun Shin"],
-      licenses: ["Apache 2.0"],
```
Comment (kianmeng) [inline mix.exs:72]: @Valian You will need to update your `maintainers` field here.

**Comment by kianmeng** [comment]
@Valian Done, please check.

**Comment by Valian** [comment]
@kianmeng LGTM, thanks! :+1:

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**